grafana/grafana/base/jupyterhub-usage-dashboard.yaml

```diff
@@ -0,0 +1,855 @@
+                "content": "The panel below displays the PVC Usage (%) for all JupyterHub users.",
+                "mode": "markdown"
+              },
+              "pluginVersion": "7.1.0",
+              "timeFrom": null,
+              "timeShift": null,
+              "title": "",
+              "type": "text"
+            },
+            {
+              "datasource": null,
+              "fieldConfig": {
+                "defaults": {
+                  "custom": {}
+                },
+                "overrides": []
+              },
+              "gridPos": {
+                "h": 3,
+                "w": 8,
+                "x": 13,
+                "y": 1
+              },
+              "id": 4,
+              "options": {
+                "content": "\nTo view your Jupyter notebook storage details in the panel below, select your user id from the `User ID` drop-down filter \non the top left of this page.\n",
+                "mode": "markdown"
+              },
+              "pluginVersion": "7.1.0",
+              "timeFrom": null,
+              "timeShift": null,
+              "title": "JupyterHub PVC Storage",
+              "type": "text"
+            },
+            {
+              "datasource": null,
+              "description": "",
+              "fieldConfig": {
+                "defaults": {
+                  "custom": {
+                    "align": "left"
+                  },
+                  "mappings": [],
+                  "thresholds": {
+                    "mode": "absolute",
+                    "steps": [
+                      {
+                        "color": "green",
+                        "value": null
+                      },
+                      {
+                        "color": "dark-orange",
+                        "value": 0.47
+                      },
+                      {
+                        "color": "semi-dark-red",
+                        "value": 1
+                      }
+                    ]
+                  },
+                  "unit": "percentunit"
+                },
+                "overrides": []
+              },
+              "gridPos": {
+                "h": 10,
+                "w": 12,
+                "x": 0,
+                "y": 3
+              },
+              "id": 6,
+              "options": {
+                "displayMode": "gradient",
+                "orientation": "horizontal",
+                "reduceOptions": {
+                  "calcs": [
+                    "mean"
+                  ],
+                  "fields": "",
+                  "values": false
+                },
+                "showUnfilled": true
+              },
+              "pluginVersion": "7.1.1",
+              "targets": [
+                {
+                  "expr": "kubelet_volume_stats_used_bytes{namespace=\"opf-jupyterhub\", pod=\"prometheus-k8s-0\"}/kubelet_volume_stats_capacity_bytes{namespace=\"opf-jupyterhub\", pod=\"prometheus-k8s-0\"}",
```

Comment:
hmm, `opf-jupyterhub` won't work for us will it?

Reply:
@anishasthana added a `namespace` template variable from which you can select the namespace you want